bundle/manifests/ocs-client-operator-config_v1_configmap.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 data:
   generateRbdOMapInfo: "true"
-  topologyFailureDomainLabels: ""
```

Comment:
why is this removed?

Reply:
Shall I add the chage mentioned in the above comment in this PR or have a new one?
`make bundle` removes this line in main branch as well